### PR TITLE
Add N.A. when no shares left

### PR DESCRIPTION
**Pull request by chhg1934**

### CODE REVIEW

File: src/GUI/ExchangeStatus.java [hunk 1]
```diff
@@ -150,7 +151,14 @@ public String toString() {
                 bestOrders.put(entry.getKey(), new EnumMap<OrderType, Double>(OrderType.class));
                 try {
                     for (Map.Entry<OrderType, SortedSet<Order>> orderEntry : exchangeServer.getExchange().getOrderStatus(security.getTicker()).entrySet()) {
-                        bestOrders.get(security).put(orderEntry.getKey(), orderEntry.getValue().first().getPrice());
+                        if(orderEntry.getValue().size()>0)
+                        {
```
Comment (aibolit) [inline src/GUI/ExchangeStatus.java]: Yo, can you please not change how the brackets are?
Alt+shift+F in Netbeans before you commit please.

**Comment by chhg1934** [comment]
Fix indentation. Alt+shift+f is really cool! :)

**Comment by aibolit** [comment]
you gotta pull first, there are conflicts

**Comment by chhg1934** [comment]
Try again. I did a git merge with your master; it should be fine now, unless I'm doing something wrong... If I am, I might need your help tomorrow. Thanks.

**Pull request closed without merging**